src/authrite.js

```diff
@@ -155,15 +173,55 @@ class Authrite {
       if (verified) {
         this.servers[baseUrl].identityPublicKey = serverResponse.identityKey
         this.servers[baseUrl].nonce = serverResponse.nonce
-        this.servers[baseUrl].requestedCertificates = serverResponse.requestedCertificates // TODO: check certs
+
+        // Check certificates were requested, and that the client is using Babbage as the signing strategy
+        if (serverResponse.requestedCertificates.certifiers.length !== 0 && this.signingStrategy === 'Babbage') {
+          // Find matching certificates
+          const matchingCertificates = await BabbageSDK.getCertificates({
+            certifiers: serverResponse.requestedCertificates.certifiers,
+            types: serverResponse.requestedCertificates.types
+          })
+
+          // IF the getCertificates function returns any certificates
+          // THEN they are added to the this.certificates within the Authrite client.
+          if (matchingCertificates.length !== 0) {
+            // Update certs to contain a keyring property
+            matchingCertificates.map(cert => {
```

Comment:
```suggestion
            matchingCertificates = matchingCertificates.map(cert => {
```
Map does not modify in place